### DISCUSSION TITLE
Place of service (service_code) requirement

### DISCUSSION THREAD

**Discussion by kavkuk** [CMSgov/price-transparency-guide#250] [Q&A]
Certain billing codes can only be performed at a specific Place of service (service_code).  If that is the case, can we use a 00 code (for 'as applicable'?) or do all the valid codes need to be listed?

(changing the service_code attribute into an array was a great update (#63 discussioncomment-1308564), but trying to narrow down the requirements for this attribute)

**Comment by perbs** [comment]
This is an interesting idea, and one I'd like to see discussed. There is a similar issue we're seeing in our development in that negotiated rates for certain procedures aren't tied to a place of service at all. These wouldn't necessarily qualify as institutional rates, though we certainly could classify them as such to make the service_code field optional. But allowing a service code of 00 would make it clear that the rates are universally applicable.

**Reply by RLTx1391** [reply]
@shaselton-usds  Any thoughts on a 00?

**Comment by shaselton-usds** [comment]
@kavkuk All relevant service codes need to be listed for the negotiated rate.

@perbs Would a [custom code for ALL service codes work](https://github.com/CMSgov/price-transparency-guide/commit/bd4b83af5977b6d5ed1f53d56f31fff3a727a4bd) for institutional rates (and possibility professional as well) address what you're asking? There has been that suggestion previously -- I'll try to find the relevant discussions to list here.

**Reply by brbucher** [reply]
Hey Scott, I think this would be [the discussion (#386)](https://github.com/CMSgov/price-transparency-guide/discussions/386) you mentioned.  Custom code for all looks great to me!

**Reply by shaselton-usds** [reply]
@brbucher Here is a recent PR: https://github.com/CMSgov/price-transparency-guide/pull/501

**Reply by bws1105** [reply]
@shaselton-usds it doesn't appear that the value of CSTM-00 for the service code has made it into the validator. Can this be added so we can pass validation using this value.

**Reply by shaselton-usds** [reply]
@bws1105 oof! The documentation of the schema was updated but not the schema itself! Here is a PR with the fix: https://github.com/CMSgov/price-transparency-guide/pull/519 . Thanks for the catch.

I'll merge it in quickly to Master and then you'll have to run an [update command on the validator](https://github.com/CMSgov/price-transparency-guide-validator#update-available-schemas) to grab the latest tag to run your file against the schema.
